src/main/java/com/exalt/resourcemanagementservice/service/ResourceService.java

```diff
@@ -0,0 +1,52 @@
+package com.exalt.resourcemanagementservice.service;
+
+import com.exalt.resourcemanagementservice.models.Request;
+import com.exalt.resourcemanagementservice.models.Server;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.concurrent.atomic.AtomicInteger;
+
+@Service
+public class ResourceService {
+    ArrayList<Server> servers=new ArrayList<Server>();
```

Comment:
how does it work in multithreading? if we have 4 requests at the same time, will have a conflict. it should be concurrent hasMap